source/features/faster-pr-diff-options.tsx

```diff
@@ -47,41 +55,41 @@ function createWhitespaceButton(): HTMLElement {
 		<a
 			href={url.href}
 			data-hotkey="d w"
-			className={`btn btn-sm btn-outline tooltipped tooltipped-s ${isHidingWhitespace ? 'bg-gray-light text-gray-light color-text-tertiary' : ''}`}
+			className={'tooltipped tooltipped-s ' + (isPRPage() ? `d-none d-lg-block color-icon-secondary ${isHidingWhitespace ? '' : 'color-icon-info'}` : `btn btn-sm btn-outline tooltipped ${isHidingWhitespace ? 'bg-gray-light text-gray-light color-text-tertiary' : ''}`)}
```

Comment:
Nested ternary operators and infinite-length line. Can be improved. I'd rather see whole strings instead of nested ones. e.g.

```js
const classes = sPRPage() ?
	'tooltipped tooltipped-s blah blah' :
	'tooltipped tooltipped-s foo foo';
```

And then

```suggestion
			className={classes}
```